Plinic/Post/PostViews/PostHeaderView.swift

```diff
@@ -13,60 +13,58 @@ struct PostHeaderView: View {
+                        .fontWeight(.bold)
+                        .padding(.all, 5)
+                        .frame(maxWidth: 300, maxHeight: 32)
+                        .background(Color.BackgroundSubColor)
+                        .foregroundColor(Color.gray)
+                        .cornerRadius(10)
+                }
+                
+                NavigationLink(destination: PostCreate(postName: "", postContext : "")){
+                    Image(systemName: "plus.circle")
+                        .font(.system(size: 35))
+                        .foregroundColor(Color(red: 0.503, green: 0.928, blue: 0.601))
                         .frame(width: 42, height: 42)
-                        .padding(.leading, 2)
-                    
-                    Button(action: {
-                        // 클릭 하면 공지 페이지로 넘어가는 기능 구현
-                    }, label: {
-                        Text("\(recentNotice.noticeTitle)")
-                            .font(.system(size: 15))
-                            .fontWeight(.bold)
-                            .padding(.all, 5)
-                            .frame(maxWidth: 300, maxHeight: 32)
-                            .background(Color.BackgroundSubColor)
-                            .foregroundColor(Color.gray)
-                            .cornerRadius(10)
-                    })
-                    
-                    NavigationLink(destination: PostCreate(postName: "", postContext : "")){
-                        Image(systemName: "plus.circle")
-                            .font(.system(size: 35))
-                            .foregroundColor(Color(red: 0.503, green: 0.928, blue: 0.601))
-                            .frame(width: 42, height: 42)
-                    }
-                    .padding(.trailing, 5)
-                } // 상단바 구현
-                .frame(maxWidth: 390, maxHeight: 46)
-                .foregroundColor(Color.MainColor)
-                .padding(.top, 2)
-            }
-            .onAppear(){
-                noticeAPI.getRecentNotice() { result in
-                    switch result {
-                    case .success(let success):
-                        self.recentNotice = success
-                        print(recentNotice)
-                    case .failure(let failure):
-                        _ = failure
-                    }
                 }
-            }
-            
+                .padding(.trailing, 5)
+            } // 상단바 구현
+            .frame(maxWidth: 390, maxHeight: 46)
+            .foregroundColor(Color.MainColor)
+            .padding(.top, 2)
         }
-        }
-        
-        struct PostHeader_Previews: PreviewProvider {
-            static var previews: some View {
-                Group {
-                    PostHeaderView()
+        .onAppear(){
+            noticeAPI.getRecentNotice() { result in
+                switch result {
+                case .success(let success):
+                    self.recentNotice = success
```

Comment:
```suggestion
                case .success(let notice):
                    self.recentNotice = notice
```
success 로 넘어온 데이터가 Notice 타입이므로 이름도 notice 로 하는게 더 코드 가독성이 좋지 않을까요?

Reply:
좋은거 같습니다! 바로 적용하겠습니다